Packs/FeedZoom/.secrets-ignore

```diff
@@ -0,0 +1,96 @@
+3.235.82.0
```

Comment:
Its better not to add that many items to the `.secrets-ignore` file.
Moreover, we are not 100% sure that they are not "clean" items.

It would be better to change the mock file ips, domains, etc.. to ones we made up (such as `www.demisto.com`, `8.8.8.8`) to be sure we are not ignoring any potentially harmful data.